content/ux-strat-bootcamp/week-one/thursday/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Thursday
+ready: true
+---
+
+# **Dashboard Visualisation and Wireframing**
+
+{{% contentlink path="ux-strat-bootcamp/week-one/thursday/data-dashboard-analysis" %}}
```

Comment:
Why so many clicks? Why not just link people straight to where they need to be instead of making them click extra?